.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
+    - run: npm run asbuild
+    - run: npm test
+      env:
+        CI: true
```

Comment:
This env variable is not used.

Reply:
It's common to set the `CI` env variable anyway — npm itself uses the variable in its usage statistics: https://docs.npmjs.com/cli/v6/using-npm/registry#does-npm-send-any-information-about-me-back-to-the-registry. It can also be used by `devDependencies`, although I don't think that's the case here.